lib/hubstats/github_api.rb

```diff
@@ -122,6 +122,25 @@ def self.update_teams
       end
     end
 
+    # Public - Goes through entire team database and updates the hubstats boolean based on the octokit.yml file
+    #
+    # Returns - nothing
+    def self.update_teams_from_file
+      team_list = Hubstats.config.github_config["team_list"] || []
+      teams = Hubstats::Team.all
+
+      teams.each do |team|
+        if (!team_list.include? team[:name]) && (team[:hubstats] == true)
```

Comment:
Should this create whitelisted teams that don't exist yet with a true boolean?  If not, the task could be more descriptive of what it does, like `deprecate_teams` instead of `update_teams_from_file`.